docs/Initial-Setup/Setting-up-your-Manager-Instance.rst

```diff
@@ -95,12 +95,16 @@ Setting up the FireSim Repo
 
 We're finally ready to fetch FireSim's sources. Run:
 
-::
+.. parsed-literal::
 
     git clone https://github.com/firesim/firesim
     cd firesim
+    # checkout latest official firesim release
```

Comment:
This comment will not make sense when looking at older versions of the doc. Maybe add a second line clarifying this may not be latest version unless you're on "latest".  Not sure it will really matter all that much.

Reply:
Ack. Added some small text. This is easy to iterate on in the future if need be.